core/execution/future/market_test.go

```diff
@@ -5648,7 +5605,7 @@ func Test2963EnsureEquityShareAreInMarketData(t *testing.T) {
 	tm.market.OnTick(ctx, tm.now)
 
 	require.NoError(t, tm.market.SubmitLiquidityProvision(ctx, lp, "party-2", vgcrypto.RandomHash()))
-	assert.Equal(t, 1, tm.market.GetLPSCount())
+	assert.Equal(t, 0, tm.market.GetLPSCount())
```

Comment:
Do you understand what happened here? Seems a bit puzzling that the LP would not be active now.

Reply:
Because we've removed liquidity monitoring, the market has left opening auction before we submit the LP, so in the liquidity engine, we return `false` for `submittedImmediately`, and so the equity shares aren't updated (which is where we get the LP count from). I could've moved around the LP submission and have `tm.market.GetLPSCount()` return the old value of 1, but I've left some comments on the spec PR, and wanted someone from research to verify this behaviour because I'm not sure what the desired behaviour here would be.  Technically, the LP submission now happens after the market leaves opening auction, but then the market is able to enter continuous trading without any provided liquidity. I'm not sure if that's entirely correct